packages/cli/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@bundler/cli",
```

Comment:
having @bundler/cli as the name will be probalamtic when publishing, shouldnt it be @map-colonies/cli ?

Reply:
we need to discuss a strategy for extending this tool, I am not aware of the future vision.
but generally, I am unattached to the name